setup.py

```diff
@@ -79,7 +79,9 @@
     # filesystem locks e.g. to prevent parallel downloads
     "filelock",
     # for fast hashing
-    "xxhash"
+    "xxhash",
+    # for better multiprocessing
+    "pathos"
```

Comment:
This is install the full pathos package and then we only use the multiprocessing part which override the default multiprocessing package, did I understand it well?